ci/ci.el7/rspec_rpmbuild/conf_files/etc/openvnet/vnmgr.conf

```diff
@@ -0,0 +1,27 @@
+#
+#       id : The ID of the OpenVNet's process. It should be unique among the entire world of the OpenVNet.
+#
+# protocol : This parameter can be used to specify the 0MQ address. Default value is 'tcp'.
+#
+#     host : Private IP address that can be used to specify the 0MQ address. 
+#
+#   public : Public/Global IP address that is linked to the private IP address specified by the 'host' parameter.
+#            A 0MQ socket will be created with the public/global IP address if this paramter is specified.
+#            Otherwise 'host' parameter will be used to create a 0MQ socket.
+#
+#     port : Listen port of the process.
+#
+#  plugins : A list of plugins for vnmgr process. vdc_vnet_plugin is used as default value but it can be blank
+#            if it does not run with Wakame-vdc.
+#
+
+node {
+  id "vnmgr"
+  addr {
+    protocol "tcp"
+    host "127.0.0.1"
+    public ""
+    port 9102
+  }
+  plugins [:vdc_vnet_plugin]
+}
```

Comment:
I know we talked about keeping these rpmbuild directories separate for different OSes but I think these config files won't change so it could still make sense to keep those in only one place. Haven't thought about which place would be good yet though.